cuchem/benchmark/scripts/benchmark.yaml

```diff
@@ -5,31 +5,34 @@ model:
 
 metric:
   validity:
-    enabled: True
+    enabled: False
     radius:
       - 1
       - 2
 
   unique:
-    enabled: false
+    enabled: False
     radius: ${metric.validity.radius}
 
   novelty:
     enabled: False
     radius: ${metric.validity.radius}
 
   nearestNeighborCorrelation:
-    enabled: false
+    enabled: False
     top_k:
       - 50
       - 100
       - 500
 
-  modelability:
-    enabled: false
+  modelabilityBioActivity:
```

Comment:
Can we make bioactivity and physchem sub parts of modelability?

Reply:
also prefer 'bioactivity' and 'physchem' in the yaml style.

Reply:
Done